pydle/features/ircv3/ircv3_2.py

```diff
@@ -39,7 +39,7 @@ class IRCv3_2Support(metadata.MetadataSupport, monitor.MonitoringSupport, tags.T
 
     async def on_capability_uhnames_available(self, value):
         """ Possibly outdated alias for userhost-in-names. """
-        return (await self.on_capability_userhost_in_names_available(value))
```

Comment:
this await is necessary as `on_capability_userhost_in_names_available` is defined as an async function as defined on l36